### ISSUE TITLE
English translation

### ISSUE THREAD

**Issue by level0dev** [frontlon/simUI-pulsar#1]
Any chance for an English version of the software?

**Comment by frontlon** [comment]
> Any chance for an English version of the software?

Yes. This software supports simplified Chinese, Traditional Chinese, English and Japanese. But other languages are translated by machines.
